sockeye/utils.py

```diff
@@ -214,6 +215,36 @@ def update(self, labels, preds):
             self.num_inst += n
 
 
+class OnlineMeanAndVariance:
```

Comment:
Lets add a link to the Wiki page where this is from.

Reply:
also, lets add some type annotations to this class and a comment that this variance version is the same as np.var